src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerAttribute.cs

```diff
@@ -39,139 +39,98 @@ public CosmosDBTriggerAttribute(string databaseName, string collectionName)
         }
 
         /// <summary>
-        /// Connection string for the service containing the collection to monitor
+        /// Gets or sets the connection string for the service containing the collection to monitor.
         /// </summary>
-        [AppSetting]
+        [ConnectionString]
```

Comment:
I should probably know this... but what's the difference between this and `[AppSetting]`?

Reply:
I actually don't know, but the `CosmosDBAttribute` file had it as `ConnectionString` so thought it should be consistent?